StripePaymentSheet/StripePaymentSheet.xcodeproj/project.pbxproj

```diff
@@ -345,6 +345,8 @@
 		A3B2F2B82CF1343F00C0E88C /* SavedPaymentMethodFormFactory+USBankAccount.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3B2F2B72CF1343F00C0E88C /* SavedPaymentMethodFormFactory+USBankAccount.swift */; };
 		A3B2F2BA2CF134B500C0E88C /* SavedPaymentMethodFormFactory+SEPADebit.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3B2F2B92CF134B500C0E88C /* SavedPaymentMethodFormFactory+SEPADebit.swift */; };
 		A3BE1C242D4161DC0061632C /* RemoveButton.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3BE1C232D4161DC0061632C /* RemoveButton.swift */; };
+		A3DA75572EB20ECE00FFFCC9 /* AttestationChallenge.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3DA75562EB20ECE00FFFCC9 /* AttestationChallenge.swift */; };
```

Comment:
You can remove these changes, the xcodeproj will now pull in source files automatically